lib/wibox/layout/fixed.lua

```diff
@@ -289,6 +289,7 @@ end
 --- Returns a new horizontal fixed layout. Each widget will get as much space as it
 -- asks for and each widget will be drawn next to its neighboring widget.
 -- Widgets can be added via :add() or as arguments to this function.
+-- Note that widgets ignore forced_height.
```

Comment:
please add the "`" around `forced_height`

Also, maybe rephrase this a little to make it clear that the widget use the preferred/minimum value in one axis and expand the other axis so the widget takes all of [that axis] space.